src/components/user-auth/questionnaire/Geolocation.tsx

```diff
@@ -0,0 +1,30 @@
+import { useGeolocation } from '@uidotdev/usehooks'
+import { Box } from '@mui/material'
+import AccessDenied from './AccessDenied'
+import { commonStyles } from 'styles/commonStyles'
+
+const Geolocation: React.FC = () => {
+  const { classes } = commonStyles()
+  const location = useGeolocation()
+
+  if (location.loading) {
```

Comment:
There is component Loader

Reply:
Not exactly "Loader", but thank you, I found it in MUI. I'm used to having a loading page or component on the project that is displayed in such cases and wanted to propose to make a task for it.